templates/http-py/metadata/snippets/component.txt

```diff
@@ -5,6 +5,6 @@ component = "{{project-name | kebab_case}}"
 [component.{{project-name | kebab_case}}]
 source = "{{ output-path }}/app.wasm"
 [component.{{project-name | kebab_case}}.build]
-command = "componentize-py -w spin-http componentize -p . -p $(python3 -c 'import site; print(site.getsitepackages()[0])') app -o app.wasm"
+command = "componentize-py -w spin-http componentize app -o app.wasm"
```

Comment:
```suggestion
command = "componentize-py -w spin-http componentize app -p . -o app.wasm"
```

We still need to include the project directory to the path I believe

Reply:
It defaults to the current directory: https://github.com/bytecodealliance/componentize-py/blob/c7b003182fc0e9ee859bb6303578640129e51cf9/src/command.rs#L61-L62

Reply:
Ah, so it needs to be specified only if multiple paths are provided!